checkbox-ng/docs/index.rst

```diff
@@ -39,17 +39,17 @@ Checkbox can be installed from a :abbr:`PPA (Personal Package Archive)`.
 
     $ sudo add-apt-repository ppa:hardware-certification/public
     $ sudo apt-get update
-    $ sudo apt-get install checkbox-ng plainbox-provider-resource-generic plainbox-provider-certification-client plainbox-provider-checkbox canonical-certification-client
+    $ sudo apt-get install canonical-certification-client
 
-The ``plainbox-provider-certification-client`` package will pull a lot of
+The ``checkbox-provider-certification-client`` package will pull a lot of
```

Comment:
Since you removed all the packages apart from canonical-certification-client in the previous command, this sentence seems out of place.